features/maintenance-mode.feature

```diff
@@ -55,3 +55,30 @@ Feature: Manage maintenance mode of WordPress install.
       """
       Error: Maintenance mode already deactivated.
       """
+
+    When I run `wp maintenance-mode activate`
+    Then STDOUT should be:
+      """
+      Enabling Maintenance mode...
+      Success: Activated Maintenance mode.
+      """
+
+    When I run `wp eval "file_put_contents('.maintenance', '<?php \$upgrading=(time()-601);'); "`
```

Comment:
Can we break this into a separate scenario?

Reply:
Sure thing.